lib/Checkout/Issuing/Controls/Create/CardControlRequest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Checkout\Issuing\Controls\Create;
+
+abstract class CardControlRequest
+{
+    public function __construct($type)
```

Comment:
I know that in all places we have public in abstract classes, but can we change to protected? Abstract classes should never be able the instantiation.

Reply:
Sure!